site/content/contribute/getting-started/contribution-checklist.md

```diff
@@ -5,30 +5,31 @@ weight: 1
 subsection: Getting Started
 ---
 
-Thanks for your interest in contributing to Mattermost! Come [join our "Contributors" community channel](https://community.mattermost.com/core/channels/tickets) on our daily build server, where you can discuss questions with community members and the Mattermost core team.
+Thanks for your interest in contributing to Mattermost! Come join our [Contributors community channel](https://community.mattermost.com/core/channels/tickets) on our daily build server, where you can discuss questions with community members and the Mattermost core team.
 
 To help with translations, [see the localization process](https://docs.mattermost.com/developer/localization.html).
 
 Follow this checklist for submitting a pull request (PR):
 
 1. You've signed the [Contributor License Agreement](http://www.mattermost.org/mattermost-contributor-agreement/), so you can be added to the Mattermost [Approved Contributor List](https://docs.google.com/spreadsheets/d/1NTCeG-iL_VS9bFqtmHSfwETo5f-8MQ7oMDE5IUYJi_Y/pubhtml?gid=0&single=true).
  - If you've included your mailing address in the signed [Contributor License Agreement](https://www.mattermost.org/mattermost-contributor-agreement/), you may receive a [Limited Edition Mattermost Mug](https://forum.mattermost.org/t/limited-edition-mattermost-mugs/143) as a thank you gift after your first pull request is merged.
-2. Your ticket is a help wanted GitHub issue for one of the Mattermost project you're contributing to.
+2. Your ticket is a Help Wanted GitHub issue for the Mattermost project you're contributing to.
     - If not, follow the process [here](/contribute/getting-started/contributions-without-ticket).
 3. Your code follows the [Mattermost Style Guide](http://docs.mattermost.com/developer/style-guide.html).
-4. Code thoroughly tested, including appropriate unit tests, [end-to-end tests for webapp](/contribute/webapp/end-to-end-tests/) and manual testing.
+4. Your code is thoroughly tested, including appropriate unit tests, [end-to-end tests for webapp](/contribute/webapp/end-to-end-tests/), and manual testing.
 5. If applicable, user interface strings are included in localization files:
     - [../mattermost-server../en.json](https://github.com/mattermost/mattermost-server/blob/master/i18n/en.json)
     - [../mattermost-webapp../en.json](https://github.com/mattermost/mattermost-webapp/blob/master/i18n/en.json)
     - [../mattermost-mobile../en.json](https://github.com/mattermost/mattermost-mobile/blob/master/assets/base/i18n/en.json)
 
     5.1 In the webapp repository run `make i18n-extract` to generate the new/updated strings.
-6. PR is submitted against `master` branch.
```

Comment:
This could be confusing, as when a PR is submitted in this case it'd prompt the user to create a new branch. Should we expand on this?

Reply:
You're correct that the user creates a branch (or a fork) for their changes - but the intent is that you should submit the changes against the `master` branch instead of, say, a release branch. Let me know if that makes sense?

Regardless, can probably be clarified

Reply:
Yep, makes perfect sense and to be fair my comment is a bit nit-picky. However, something like "PR is submitted against `master` branch in a new fork (as prompted in the GitHub UI)" might work?

Reply:
Yeah that works for me 👍